src/05_output_LFS.R

```diff
@@ -0,0 +1,99 @@
+# this file is used for outputting data for importing to DIP
```

Comment:
@thister all code files need to include the Apache 2.0 boilerplate header to meet the licence requirements. You can install the `bcgovr` package from bcgov GitHub and add the snippet at the top of the script with an RStudio addin.
![Screenshot 2024-11-15 at 3 42 56 PM](https://github.com/user-attachments/assets/07b89049-ec09-4f0a-9dea-136fc526f8f1)

Reply:
done!